### DISCUSSION TITLE
OQS_MEM_insecure_free function showing not allocated in liboqs.a file

### DISCUSSION THREAD

**Discussion by Jpgig** [open-quantum-safe/liboqs#1394] [General]
I compiled liboqs as per instructions given in the readme file but generated output showing OQS_MEM_insecure_free as an undefined symbol.
 ![image](https://user-images.githubusercontent.com/115164852/219795901-47cdae96-352f-4656-aaad-2b3a98153c12.png)
I check the usage of this function and it has been used in many places still symbol is not get generated in the output file.

Any suggestion to get a symbol assigned in the output file?

**Comment by baentsch** [comment]
The symbol looks pretty defined to me ("T"). What's your underlying/original problem? Please provide detail information as to which platform/tools are you using.

**Comment by Jpgig** [comment]
I am integrating this version of openssl with the python cryptography module. I am able to integrate it now and look to access the PQC function with the cryptography module.

I found below cryptography folder under quantum-safe openssl git repo:
![image](https://user-images.githubusercontent.com/115164852/221279833-d678e814-d23a-4c25-8888-fada40978673.png)
does this mean the quantum-safe openssl version is already integrated into the python cryptography module?

**Reply by dstebila** [reply]
No, this means that pyca-cryptography is a dependency of OpenSSL.
